data/image_loader.py

```diff
@@ -0,0 +1,9 @@
+import cv2
+
+
+class ImageLoader:
+
+    @staticmethod
```

Comment:
Usuń staticmethod i zrób z tego dependency injection, czyli w MapInitializer __init__ dodaj imageloader: ImageLoader = ImageLoader()

Reply:
Nie wiem czy o to chodziło, ale zmieniłem